captura/views.py

```diff
@@ -224,7 +227,153 @@ def create_estudio(request):
     if request.method == 'POST':
         familia = Familia.objects.create()
         Estudio.objects.create(capturista=request.user.capturista, familia=familia)
-        return redirect(reverse('home'))
+        return redirect(reverse('captura:familia', kwargs={'id_familia': familia.id}))
+
+
+@login_required
+@user_passes_test(is_capturista)
+def familia(request, id_familia):
```

Comment:
Also, I think this isn't finished until we find a way to make the flow more natural: right now if you try to edit the family you have to complete this akward two step process: first filling the civil state and so on, and just then going to the member section. 

Maybe this could be improved by removing the edit button altogether and just be able to edit some fields when we see the detail of a study?
this is open for discussion @fernandolobato but can't approve this w/o checking into the matter.